packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -103,7 +97,10 @@ export class GadgetConnection {
     } else if (typeof window != "undefined" && window.fetch) {
       this._fetchImplementation = window.fetch.bind(window);
     } else {
-      this._fetchImplementation = fetchPolyfill;
+      this._fetchImplementation = async (...args: [any]) => {
```

Comment:
nit

```suggestion
      this._fetchImplementation = async (...args: any[]) => {
```

Reply:
Fun fact: typescript complains if you don't use a tuple type for the args, that's why its like this

Reply:
TIL 🧠